content/en/tracing/guide/trace_ingestion_volume_control.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Ingestion volume control with APM Distributed Tracing
+kind: guide
+further_reading:
+- link: "/tracing/trace_ingestion/ingestion_controls/"
+  tag: "Documentation"
+  text: "Ingestion Control Page"
+---
+
+## Overview
+
+The [Ingestion control page][1] provides granular visibility over the ingestion configuration for all services, in the agent and in the tracing libraries. All [Ingestion Mechanisms][2] are publicly documented and configurable.
+
+With the page, you have full visibility and control your span volume. Consequently, you are be able to:
+- Ingest the data that is most relevant to your business and your observability goals.
+- Reduce network costs by avoiding sending unused trace data to the Datadog platform.
+- Control and manage your overall costs
+
+
+## What happens if I decide to reduce my ingested trace volume?
+
+Sampling does not mean less value out of Datadog APM
+
+{{< img src="/tracing/guide/trace_ingestion_volume_control/sampling_25_percent.png" alt="APM ingestion sampling" style="width:70%;" >}}
+
+Even by reducing the ingestion volume for certain services, the **request, error, and latency metrics** (RED metrics) will remain 100% accurate as they are being calculated based on 100% of the applications’ traffic, regardless of any sampling configuration (i.e. this is done by a different system). These metrics are included when purchasing Datadog APM. In order to make sure you have full visibility into your application’s traffic, you can use these metrics to spot potential errors on a service or a resource, by creating monitors and SLOs.
+
+Then, because trace data is very repetitive, you will have enough trace samples to investigate any issues. For high throughput services, there should not be any need for you to collect every single request - an important enough problem will always show symptoms in multiple traces. Datadog puts the control in your hands, allowing you to have the visibility that you need to solve problems while remaining within budget.
```

Comment:
```suggestion
Then, because trace data is very repetitive, you should have enough trace samples to investigate any issues. For high throughput services, there should not be any need for you to collect every single request - an important enough problem should always show symptoms in multiple traces. Datadog puts the control in your hands, allowing you to have the visibility that you need to solve problems while remaining within budget.
```